### DISCUSSION TITLE
More Examples for `tokio_utils` crate

### DISCUSSION THREAD

**Discussion by artech-git** [tokio-rs/tokio#6383] [General]
After working with tokio for a long time, I feel there needs to be more examples around usage of `tokio-utils` especially about how to use Framed data and `Encoder`/`Decoder` traits.

**Comment by Darksonn** [comment]
There have actually been some changes to the docs. See https://github.com/tokio-rs/tokio/pull/6310. There just hasn't been a release of tokio-util release since then.

If you have further suggestions, then please open a documentation bug instead of a discussion. Discussions are intended for help — not bugs or feature requests.
